pkg/pillar/cmd/volumemgr/blob.go

```diff
@@ -88,8 +88,11 @@ func downloadBlob(ctx *volumemgrContext, blob *types.BlobStatus) bool {
 	if ds.HasError() {
 		log.Errorf("Received error from downloader for blob %s: %s",
 			blob.Sha256, ds.Error)
-		blob.SetErrorWithSource(ds.Error, types.DownloaderStatus{},
-			ds.ErrorTime)
+		description := ds.ErrorDescription.Copy()
+		if description.ErrorRetryCondition == "" {
+			description.ErrorRetryCondition = ds.Error
```

Comment:
Would would the retrycondition be the same as the error?

Reply:
Good point, will jump deeper into the tree, now it looks like they all are repeatable, but let me check.

Reply:
My point is that the downloader should set that; here you just propagate the error (and you might add some entities as part of the propagation, but not change anything else)

Reply:
Now it is inside `func (status *DownloaderStatus) HandleDownloadFail`